dev-utils/src/commands/genesis_dev_config.rs

```diff
@@ -0,0 +1,55 @@
+use chain_core::init::config::InitialValidator;
+use chain_core::init::{address::RedeemAddress, coin::Coin};
+use chrono::offset::Utc;
+use chrono::DateTime;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+use std::str::FromStr;
+
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
does it need "Serialize"?

Reply:
for writing, it's necessary

Reply:
where does one need to write it?

Reply:
InitCommand::new()
 data: serde_json::to_string(&GenesisDevConfig::new()).unwrap(),

Reply:
https://github.com/crypto-com/chain/pull/238/files#diff-11498f5b59098eafc02087e776f6782cR59 -- but that `data` field is subsequently again parsed back to the original struct

Reply:
agree, i'll remove data json string, will use Genesis class directly

Reply:
fixed